README.md

```diff
@@ -1,5 +1,12 @@
 # catalogd
 
+---
+> [!WARNING]  
+> Since release 4.18 and January 2025, Catalogd's functionality has been integrated into v1's primary repository: [operator controller](https://github.com/openshift/operator-framework-operator-controller).
```

Comment:
```suggestion
> Since release 4.18 and January 2025, Catalogd's functionality has been integrated into OLMv1's primary repository: [operator controller](https://github.com/openshift/operator-framework-operator-controller).
```

Reply:
It was done I forgot to push :-(